packages/core/src/node/config/getNodeJestConfig.ts

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+
+export function getNodeJestConfig() {
+  return {
+    displayName: {
+      color: 'redBright',
+      name: 'node',
+    },
+
+    resolver: 'jest-directory-named-resolver',
+    setupFilesAfterEnv: [path.resolve(__dirname, './jest.setup.node.js')],
```

Comment:
Could we look for a jest setup file in `<baseDir>/<side>/config/jest.setup.js`?

Reply:
Yep! Good one.

Reply:
Actually, nope!

If you list a file in this array Jest doesn't just look for it, it errors when it's not there.

It's very easy for a user to add their own tho, they could just do:
```js
config.setupFilesAfterEnv.push('./jest.setup.js')
```

We can add some docs on this.